var/spack/repos/builtin/packages/cgal/package.py

```diff
@@ -85,7 +85,9 @@ def install(self, spec, prefix):
         else:
             options.append('-DBUILD_SHARED_LIBS:BOOL=OFF')
 
-        cmake('.', *options)
-
-        make()
-        make('install')
+        build_directory = join_path(self.stage.path, 'spack-build')
+        source_directory = self.stage.source_path
+        with working_dir(build_directory, create=True):
+            cmake(source_directory, *options)
+            make()
+            make('install')
```

Comment:
Why not:
```python
with working_dir('spack-build', create=True):
    cmake('..', *options)

    make()
    make('install')
```
No need to use absolute paths. This makes it simpler, and it's the default when you create a new CMake package with `spack create`